packages/jsonld/tests/signature.test.ts

```diff
@@ -0,0 +1,40 @@
+import { signRsaSignature, verifyRsaSignature, generateRsaKeyPair } from '../src';
```

Comment:
Why did you name it `tests/signature.test.ts`? I would expect something like `tests/signature.ts`

Reply:
I'll suggest keeping the `signature.test.ts` because it's Jest test file naming convention.

Reply:
It looks redundant. Can you point out similar TypeScript-based projects which follow this convention?

Reply:
iota.js
https://github.com/iotaledger/iota.js/tree/next/packages/bundle/test
Canner
https://github.com/Canner/canner/tree/canary/packages/canner-compiler/test